src/services/InstantAnswerService.ts

```diff
@@ -17,28 +17,32 @@ class InstantAnswerService {
 		return this.instance;
 	}
 
-	async query(query: string): Promise<InstantAnswer> {
+	async query(query: string): Promise<InstantAnswer|null> {
```

Comment:
Can you put a space on either side of the pipe, please? Reads better in my opinion.